dcc-repository-resources/src/main/resources/org/icgc/dcc/repository/resources/mappings/donor-text.mapping.json

```diff
@@ -31,6 +30,7 @@
                   "index":"analyzed",
                   "index_analyzer":"id_index",
                   "search_analyzer":"id_search",
+                  "analyzer": "id_search",
```

Comment:
Elasticsearch will throw an exception saying: 
`analyzer on field [xxx] must be set when search_analyzer is set`